OWNERS_ALIASES

```diff
@@ -2,39 +2,27 @@
 
 aliases:
   sig-cluster-lifecycle-leads:
-    - lukemarsden
+    - justinsb
     - luxas
-    - roberthbailey
     - timothysc
   cluster-api-admins:
+    - detiber
     - justinsb
-    - krousey
-    - luxas
-    - roberthbailey
-    - kris-nova
```

Comment:
Update to sync with https://github.com/kubernetes-sigs/cluster-api/pull/884

Reply:
@detiber This should be updated to add dwat